### DISCUSSION TITLE
Remote Ollama server: Model Selection Issue

### DISCUSSION THREAD

**Discussion by SamuelDevdas** [open-webui/open-webui#2372] [General]
### Discussed in https://github.com/open-webui/open-webui/discussions/2359

<div type='discussions-op-text'>

<sup>Originally posted by **SamuelDevdas** May 17, 2024</sup>
# Bug Report

## Description

**Bug Summary:**
[When using ollama installed on remote server using docker, how to select models?(
![Screenshot 2024-05-17 225548](https://github.com/open-webui/open-webui/assets/84823680/bee13746-d26f-42a8-be68-ccdb82b9c46b)
)]
![WhatsApp Image 2024-05-17 at 23 00 55_99cf826f](https://github.com/open-webui/open-webui/assets/84823680/fb0068aa-c518-4d7d-a351-c5d2762ff7b2)

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Available models should be listed, or an options while setting up docker container should be available]

**Actual Behavior:**
[Models not available to select, thus chat cant be started]

## Environment

- **Open WebUI Version:** [e.g., 0.1.120]
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** [raspbian]
- **Browser (if applicable):** [e.g., Chrome 100.0]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!
</div>

**Comment by Szymok** [comment]
Did you resolved the issue?

**Comment by BoneGoat** [comment]
Just ran into the same issue. Worked around it by going into the open-webui admin settings and manually downloading a model. When that's complete I can select it and start a conversation.
